data/processSynapseJSON.ts

```diff
@@ -67,9 +65,38 @@ function addReleaseInfo(
     }
 }
 
-function addDownloadSourcesInfo(file: BaseSerializableEntity) {
-    const dbgapSynapseSet = new Set(dgbapIds);
-    const dbgapImgSynapseSet = new Set(dbgapImageIds);
+function getDbgapSynapseIds(entitiesById: {
+    [entityId: string]: ReleaseEntity;
+}) {
+    return getReleaseSynapseIds(
```

Comment:
Not sure if it's critical, but number of unique ids we get from `entities_v4.csv` does not match the number of unique ids in `dbgap_release_all.json`